test/SymbolGraph/ClangImporter/ForeignExtensions.swift

```diff
@@ -0,0 +1,19 @@
+// RUN: %empty-directory(%t)
+// RUN: cp -r %S/Inputs/EmitWhileBuilding/EmitWhileBuilding.framework %t
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -enable-objc-interop -emit-module-path %t/EmitWhileBuilding.framework/Modules/EmitWhileBuilding.swiftmodule/%target-swiftmodule-name -import-underlying-module -F %t -module-name EmitWhileBuilding -disable-objc-attr-requires-foundation-module %s %S/Inputs/EmitWhileBuilding/Extra.swift -emit-symbol-graph -emit-symbol-graph-dir %t
```

Comment:
Should we also have a test that checks the behavior when using `swift-symbolgraph-extract`, since the behaviors regarding `@_export import` are different?

Reply:
That makes sense! I'll write up a test and push it today.